### PR TITLE
Add leak map to landing page

### LINKED ISSUE #5: Add gas leak mapping to the landing page

### PR DESCRIPTION
**Pull request by kevincianfarini**
Closes #5

### CODE REVIEW

**Comment by kevincianfarini** [comment]
image

File: docs/index.md [hunk 1]
```diff
@@ -9,8 +9,8 @@ under neighborhoods throughout the city and counties. Some pipes are 100 years o
 pipes are decayed and leaking, and nearly a thousand gas leaks require fixes by Richmond's Department of Public 
 Utilities. Dozens of these leaks are liable to ignite or explode. 
 
-`INSERT LEAK MAP`
-
+{% include leak-map.html %}
+\
```
Comment (kevincianfarini) [inline docs/index.md:13]: This backslash is a markdown line break which helps some of the padding below the map.

Comment (eshilts) [reply docs/index.md:13]: Good to know

File: docs/_includes/leak-map.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<style>.embed-container {position: relative; padding-bottom: 100%; height: 0; max-width: 100%;} .embed-container iframe, .embed-container object, .embed-container iframe{position: absolute; top: 0; left: 0; width: 100%; height: 100%;} small{position: absolute; z-index: 40; bottom: 0; margin-bottom: 0;}</style><div class="embed-container"><iframe width="800" height="400" frameborder="0" scrolling="no" marginheight="0" marginwidth="0" title="Public Methane Leak Map" src="//urichmond.maps.arcgis.com/apps/Embed/index.html?webmap=acecca1c02bf478ab965196eb6d00ee1&extent=-77.6997,37.4206,-77.1927,37.6408&home=true&zoom=true&previewImage=false&scale=true&search=true&searchextent=true&legend=true&disable_scroll=true&theme=light"></iframe></div>
```
Comment (eshilts) [inline docs/_includes/leak-map.html:1]: Good job putting it as a fragment so it can be embedded anywhere.

**Review by eshilts — APPROVED** [review]
This is dope

**Pull request merged**